deploy/deploy_research_environment.yml

```diff
@@ -38,6 +40,43 @@
         dest: "{{ item.dest }}"
       with_items: "{{ printers_symlinks }}"
 
+    - name: additional symlinks for odbc.ini, odbcinst.ini, freetds, cups to shared env
+      file:
+        state: link
+        src: "{{ item.src }}"
+        dest: "{{ item.dest }}"
+      with_items: "{{ research_environment_links }}"
+
+    - name: Add /etc/devfs.rules file to research env servers
+      template:
+        src: devfs.rules
+        dest: /etc/devfs.rules
+        mode: 0644
+
+    - name: Add rc.conf file to all research env servers
+      template:
+        src: rc.conf    
+        dest: /etc/rc.conf 
+        mode: 0644
+
+    - name: Install postgresq packages in research environment
+      yum:
+        name: Install postgresql packages 
+        state: present  
+        disable_gpg_check: yes 
+      with_items:
+        - postgresql95
+        - postgresql95-devel
```

Comment:
@olivermel These two packages should be removed, and remain in the postgresql-client role.